Parse/Parse.csproj

```diff
@@ -142,9 +141,5 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-  <ItemGroup>
-    <Folder Include="Internal\Config\" />
-    <Folder Include="Internal\Push\State\" />
-    <Folder Include="Internal\Push\Coder\" />
-  </ItemGroup>
+  <ItemGroup />
```

Comment:
I think it was actually generated by MonoDevelop. I'll try removing it and see how much breaks.